pubspec.yaml

```diff
@@ -88,3 +91,7 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+  fonts:
+    - family: Inter
+      fonts:
+        - asset: fonts/Inter-VariableFont_slnt,wght.ttf
```

Comment:
Why not name the font file better than what you have here?

Reply:
Generally to get different font weights (or slant angles), we rely on different font files. The font file used here supports variable font slant and weights. Hence the default naming of it and I didn't change it.
Rather it would be better to mention this functionality as a comment in pubspec file, but change the complex name of the font file.

Reply:
No, I mean why not at least be consistent with your usages of `_` and `-` and name the file well (use `slant` rather than `slnt`)?

Reply:
Ok. Got your point. It was about the consistency in naming. I though of it as an issue with the jargons in the name file. Renaming the file again.